src/OrbitGl/PrimitiveAssemblerTest.cpp

```diff
@@ -18,6 +18,8 @@ const Vec2 kTopRight{5, 0};
 const Vec2 kBottomRight{5, 5};
 const Vec2 kBottomLeft{0, 5};
 const Vec2 kBoxSize{5, 5};
+const Vec2 kArrowSize{2, 4};
+const Vec2 kArrowHeadSize{3, 2};
```

Comment:
It's a bit strange that the `arrow_head` is bigger than the `total_arrow_size`. As I told you above, consider instead having `body_size` and `head_size`